pandas/tests/io/pytables/test_append.py

```diff
@@ -437,28 +437,28 @@ def check_col(key, name, size):
 
     with ensure_clean_store(setup_path) as store:
 
-        def check_col(key, name, size):
+        def check_col_1(key, name, size):
             assert getattr(store.get_storer(key).table.description, name).itemsize, size
```

Comment:
Thanks for doing this @FloHofstetter !

🤔 what an odd function...I wonder if it was meant to be
```
            assert getattr(store.get_storer(key).table.description, name).itemsize == size
```

I'll look through the history to see if there's any clues

Reply:
Indeed 😄 

https://github.com/pandas-dev/pandas/pull/16169/files

```diff
             def check_col(key, name, size):
-                self.assertEqual(getattr(store.get_storer(
-                    key).table.description, name).itemsize, size)
+                assert getattr(store.get_storer(key)
+                               .table.description, name).itemsize, size
```
cc @gfyoung this is typo right? Just double-checking, in case you remember

Looks like this should be the same as `check_col_0` above, can we just keep that one as `check_col` and remove this one @FloHofstetter ?

Reply:
Yep, that indeed looks like a typo @MarcoGorelli